lib/holidays.rb

```diff
@@ -122,6 +122,40 @@ def between(start_date, end_date, *options)
       UseCaseFactory.between.call(start_date, end_date, date_driver_hash, regions, observed, informal)
     end
 
+    # Get next holidays occuring form date, inclusively.
+    #
+    # Returns an array of hashes or nil.
+    #
+    # Each holiday is returned as a hash with the following fields:
```

Comment:
Oh , i have written wrong comments. 😭 
Fields are incoming arguments.
I modify  this line comment.